cartridges/app_custom_cartridge/steptypes.json

```diff
@@ -0,0 +1,33 @@
+{
+    "step-types": {
+        "script-module-step": [
+            {
+                "@type-id": "custom.checkProductInfo",
```

Comment:
The job is exporting product id's, the name of the job should not be 'checkProductInfo' then, please use appropriate name